src/common/linebuf-reader.h

```diff
@@ -81,6 +81,8 @@ class LinebufReader {
   const char *const buffer_end_;
   char *content_start_;
   char *content_end_;
+
+  enum CH_STATE { IN_LINE, CR_SEEN, LF_SEEN } ch_state_;
```

Comment:
To make the name a little more expressive. Maybe `parse_state_` ?

Reply:
(solved, see below)